stylesheets/Components/modal.css

```diff
@@ -0,0 +1,19 @@
+.modal-container{
+    max-width: 50%;
+    margin: auto;
+    background-color: var(--st-light-pink);
+    padding: 15px;
+}
+.modal-info, .modal-btn{
+    padding: 20px;
+}
+.btn-modal-solid{
+    padding: 15px;
```

Comment:
try to use rems

Reply:
root-font size is 20px..... and i also don't want the padding to be changed as it was looking odd when i used rems.